templates/ec2/sc-ec2-linux-docker-notebook.yaml

```diff
@@ -358,7 +358,7 @@ Resources:
                   'Fn::FindInMap': [NotebookTypeToInitParams, !Ref NotebookType, EC2WorkDir]
                 DOCKER_IMAGE:
                   'Fn::FindInMap': [NotebookTypeToInitParams, !Ref NotebookType, DockerImage]
-                JUPYTER_WORK_DIR: /home/jovyan/workdir
+                JUPYTER_WORK_DIR: /home/jovyan/work
```

Comment:
might be better to give this a generic name, like `APP_WORK_DIR` and add it to the Jupyter and Rstudio mappings?